src/v2/Components/v2/ArtworkFilter/ArtworkFilterMobileActionSheet.tsx

```diff
@@ -19,11 +20,35 @@ export const ArtworkFilterMobileActionSheet: SFC<{
     initialArtworkFilterState
   )
 
+  useEffect(() => {
+    // While mobile sheet is mounted, the effect of the user's filter selections
+    // should be deferred until the Apply button is pressed, rather than
+    // applied immediately. Therefore…
+    //
+    // On mount, enter deferred mode, and initialize a set of deferred filter
+    // changes from the current filter choices.
+    filterContext.toggleDeferFilterChanges(true)
+    filterContext.setDeferredFilters(filterContext.filters)
+
+    // On unmount, exit deferred mode.
+    return () => {
+      filterContext.toggleDeferFilterChanges(false)
+    }
+  }, [])
+
   return (
     <Container>
       <Header p={1}>
-        <Button variant="noOutline" size="small" onClick={onClose}>
-          Close
+        <Button
+          variant="noOutline"
+          size="small"
+          onClick={() => {
+            // On close, abandon any deferred filter changes
+            filterContext.setDeferredFilters({})
```

Comment:
@anandaroop - this implementation is looking really good! 👌 

One q around naming -- why "deferrered" over "staged"? Pitching in for "staged", it follows a lot of other similar concepts that coders will be familar with (in particular git). It also feels a bit more generic. Then different actions can read more declaratively -- "stage filters", "unstage filters", etc.